scalameta/scalameta/src/test/scala/scala/meta/tests/parsers/ModSuite.scala

```diff
@@ -239,9 +239,39 @@ class ModSuite extends ParseSuite {
     )
   }
 
+  test("abstract override") {
+    /* Non-trait members modified by `abstract override` receive a typechecking error */
```

Comment:
Typechecking, or even in `RefChecks`. Scalac swallows almost all the `override` modifiers and performs other checks in those aforementioned phases.